torch_tvm/custom_tvm_ops/cpp/topi/contrib/quantize.cc

```diff
@@ -20,18 +20,31 @@ TVM_REGISTER_GLOBAL("tvm.contrib.find_minmax")
       CHECK(input->ndim == 2) << "find_minmax only support the two dimenstion input";
       int m = input->shape[0];
       int n = input->shape[1];
-      float d_min = data_ptr[0];
-      float d_max = data_ptr[0];
+      int num_els = m * n;
+      int num_iters = num_els / 16;
+      int num_left_overs = num_els % 16;
+      float min_v[16] = {0.f};
+      float max_v[16] = {0.f};
       for (int i = 0; i < m; i++) {
-        for (int j = 0; j < n; j++) {
-            d_min = std::min(data_ptr[i*n + j], d_min);
-            d_max = std::max(data_ptr[i*n + j], d_max);
+        for (int j = 0; j < 16; j++) {
+            min_v[j] = std::min(data_ptr[i*16 + j], min_v[j]);
+            max_v[j] = std::max(data_ptr[i*16 + j], max_v[j]);
```

Comment:
Can you give a slow path for n less than 16?

Reply:
This covers should cover all shapes. However I need to initialize min and max appropriately. 0 is not the right value for those.

Reply:
@llyfacebook, fixed the bug.